docs/source/tutorial.rst

```diff
@@ -178,10 +188,23 @@ Let’s see what type of value we get for different integrators.
 
 .. parsed-literal::
 
-    **Output:** Results: 12.778112411499023
-            Abs. Error: 0.00000000e+00
-            Rel. Error: 0.00000000e+00
+    **Output:** Results: (0.0748857706785202-1.4073622226715088j)
+            Abs. Error: 1.21286661e-07
+            Rel. Error: 8.60583995e-08
     
+.. code:: ipython3
+
+    boole = Boole()
+    result = boole.integrate(f, dim=1, N=101, integration_domain=integration_domain)
+    print_error(result,solution)
+
+
+.. parsed-literal::
+
+    **Output:** Results: (0.0748857855796814-1.4073619842529297j)
+            Abs. Error: 1.19441893e-07
+            Rel. Error: 8.47494519e-08
```

Comment:
Here we should either:
- Reduce N to have differences between integrators
- Switch to double precision to have differences between integrators
- Mention that the numerical error is within range of floating point precision and hence no improvement can be seen for Boole